### PR TITLE
[#185797525] - switch to using trino internal uri/trino client for e2e tests

### PR DESCRIPTION
**Pull request by jabran-khan**
Link to [ticket](#185797525)

### CODE REVIEW

File: helm/templates/tests/e2etests.yaml [hunk 1]
```diff
@@ -62,14 +62,22 @@ spec:
...
             - name: E2E_PUBLISHER_DATA_RESOURCE_URI
               value: '{{- include "dnastack.publisherData.resourceUrl" . -}}/'
-            - name: DD_CIVISIBILITY_ENABLED
-              value: "true"
-            - name: DD_ENV
-              value: "dnastack-us-central-cluster_{{ coalesce .Values.cloud.clusterNamespace ((.Values.global).cloud).clusterNamespace }}"
-            - name: DD_SERVICE
-              value: "{{ .Values.app.name }}"
-            - name: DD_TRACE_AGENT_URL
-              value: "http://10.31.241.233:8126"
```
Comment (Kevin-DNA) [inline helm/templates/tests/e2etests.yaml:72]: Should these be removed? I think @prajjwolmondal is still working on adding test monitoring. I think the better place to add it (if you remove it from here) would be in the optional test.env section in the values file but it's not there either.

Comment (usanthan) [reply helm/templates/tests/e2etests.yaml:72]: Yes Jabran and I handled that as part of this commit: https://github.com/DNAstack/cloud-deployments/commit/9fc11f894f7202be288a216b8102afd90a51cd77

File: e2e-tests/src/main/java/com/dnastack/ga4gh/dataconnect/adapter/DataConnectE2eTest.java [hunk 2]
```diff
@@ -273,49 +263,51 @@ private static void setupTestTables() {
...
-                }
-            });
-        } catch (SQLException se) {
-            log.error("Error connecting to the server.  SQL State: {}\n{}", se.getSQLState(), se.getMessage());
-            throw new RuntimeException("Unable to setup test tables.", se);
+    private static void waitForQueryToFinish(String query) throws IOException, InterruptedException {
+        JsonNode node = trinoHttpClient.query(query, Map.of());
+        String state = node.get("stats").get("state").asText();
+        while (!state.equals("FINISHED")) {
+            Thread.sleep(1000);
```
[the remark this answers is not in the document]
Comment (usanthan) [reply e2e-tests/src/main/java/com/dnastack/ga4gh/dataconnect/adapter/DataConnectE2eTest.java:285]: I noticed trino would error sometimes if I didn't have some delay between fetching the next page

File: helm/templates/deployment.yaml [hunk 3]
```diff
@@ -153,7 +153,7 @@ spec:
               path: /actuator/health
               port: 8080
               scheme: HTTP
-            initialDelaySeconds: 610
+            initialDelaySeconds: 10
```
[the remark this answers is not in the document]
Comment (prajjwolmondal) [reply helm/templates/deployment.yaml:156]: Oh wow!

**Pull request merged**